devices/src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub mod legacy;
 pub mod virtio;
```

Comment:
nit: typo in commit message c8e8868e133db4df3b3effd6fcb5a6002e2bbe3f - s/actuall/actual

Reply:
Fixed